portfolio/src/main/webapp/form.html

```diff
@@ -0,0 +1,60 @@
+<html>
+<head>
+  <meta charset="UTF-8">
+   <link rel="stylesheet" href="formStyle.css">
+   <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">  
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    
+
+</head>
+<body>
+
+<h2>Contact Me</h2>
+
+
+<div id="container">
+  <form action="/form-handler" method = "POST">
+  <div class="row">
+    <div class="col-25">
+      <label for="fname">First Name</label>
+    </div>
+    <div class="col-75">
+      <input type="text" id="fname" name="first-name" placeholder="Your name..">
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-25">
+      <label for="lname">Last Name</label>
+    </div>
+    <div class="col-75">
+      <input type="text" id="lname" name="last-name" placeholder="Your last name..">
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-25">
+      <label for="country">Country</label>
+    </div>
+    <div class="col-75">
+      <select id="country" name="country">
+        <option value="australia">Australia</option>
+        <option value="canada">Canada</option>
+        <option value="usa">USA</option>
+      </select>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-25">
+      <label for="subject">Subject</label>
+    </div>
+    <div class="col-75">
+      <textarea id="subject" name="message" placeholder="Write something.." style="height:200px"></textarea>
```

Comment:
We want style to always go in the CSS file when you can because this will help keep things orderly, and also because inline CSS overrides the CSS file, so this way we don't have to deal with confusing instances.